src/SFML/Window/Win32/WglContext.cpp

```diff
@@ -88,7 +89,14 @@ namespace priv
 String getErrorString(DWORD errorCode)
 {
     PTCHAR buffer;
-    if (FormatMessage(FORMAT_MESSAGE_MAX_WIDTH_MASK | FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS, nullptr, errorCode, 0, reinterpret_cast<LPTSTR>(&buffer), 256, nullptr) != 0)
+    if (FormatMessage(FORMAT_MESSAGE_MAX_WIDTH_MASK | FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM |
+                          FORMAT_MESSAGE_IGNORE_INSERTS,
```

Comment:
_Might just be me being pedantic_ The formatting here looks out of place with the section below it, I can understand why but it did catch my eye

Reply:
> There's also the [--exit-code argument](https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---exit-code), which returns 0 if no differences, and 1 if differences are found.

That is correct. See the [BinPackParameters](https://releases.llvm.org/12.0.0/tools/clang/docs/ClangFormatStyleOptions.html) option. I prefer we not use this, but it does exist.